eng/pipelines/global-build.yml

```diff
@@ -76,4 +76,19 @@ jobs:
     jobParameters:
       testGroup: innerloop
       nameSuffix: RuntimeFlavor_Mono
-      buildArgs: /p:RuntimeFlavor=Mono
\ No newline at end of file
+      buildArgs: /p:RuntimeFlavor=Mono
+
+#
+# Build Mono + Libraries. This excercises the code path where we build libraries without
+# first building CoreCLR
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: debug
+    platforms:
+    - Windows_NT_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: Runtime_Release
```

Comment:
I think you need to change this or otherwise it'll conflict with the other job using `nameSuffix: Runtime_Release`